CMakeLists.txt

```diff
@@ -14,6 +14,20 @@ endif()
 cmake_policy(SET CMP0063 NEW)
 cmake_policy(SET CMP0074 NEW)
 
+# https://reproducible-builds.org/docs/source-date-epoch/
```

Comment:
@cyyever I used the variant posted at "https://reproducible-builds.org/docs/source-date-epoch/" described as "If you would like to support legacy/archival versions of CMake, you can use this less-preferred variant:" =>If our minimum required cmake variant were newer, perhaps it could be adapted.